test_infra/README.md

```diff
@@ -0,0 +1,62 @@
+
```

Comment:
Do we need both this README and the contributing guide?

Reply:
Nope, will remove.